### PR TITLE
fixed #26

### LINKED ISSUE #26: Fix vulnerabilities
Failed to execute goal org.sonatype.ossindex.maven:ossindex-maven-plugin:3.2.0:audit (default-cli) on project bucketfs-document-files-virtual-schema: Detected 3 vulnerable components:
* [org.apache.hadoop:hadoop-common:jar:3.3.4](https://ossindex.sonatype.org/component/pkg:maven/org.apache.hadoop/hadoop-common@3.3.4?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1) in compile
    * [sonatype-2022-5820](https://ossindex.sonatype.org/vulnerability/sonatype-2022-5820): 1 vulnerability (8.2)
* [org.apache.hadoop:hadoop-hdfs-client:jar:3.3.4](https://ossindex.sonatype.org/component/pkg:maven/org.apache.hadoop/hadoop-hdfs-client@3.3.4?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1) in compile
    * [sonatype-2022-5732](https://ossindex.sonatype.org/vulnerability/sonatype-2022-5732): 1 vulnerability (8.6)
* [com.fasterxml.jackson.core:jackson-databind:jar:2.13.4](https://ossindex.sonatype.org/component/pkg:maven/com.fasterxml.jackson.core/jackson-databind@2.13.4?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1) in compile
    * CVE-2022-42003, severity CWE-502: Deserialization of Untrusted Data (7.5)
  
Additionally check, if currently the following vulnerability still needs to remain excluded: 
    - 1 vulnerability found (6.2); https://ossindex.sonatype.org/vulnerability/sonatype-2020-0926

### PR DESCRIPTION
**Pull request by ckunki**

### CODE REVIEW

File: doc/changes/changes_1.0.1.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Virtual Schema for document data on BucketFS 1.0.1, released 2022-??-??
```
Comment (redcatbear) [inline doc/changes/changes_1.0.1.md:1]: Don't forget to update the release date.

File: doc/changes/changes_1.0.1.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Code name: Dependency update
```
Comment (redcatbear) [inline doc/changes/changes_1.0.1.md:3]: ... on top of 1.0.0

(add for disambiguation)

File: pom.xml [hunk 3]
```diff
@@ -162,8 +162,15 @@
                 <artifactId>ossindex-maven-plugin</artifactId>
                 <configuration>
                     <excludeVulnerabilityIds>
-                        <!-- com.google.guava:guava:31.1-jre: Excluded because we assume that the attacker does not have access to the machine. -->
+                        <!-- com.google.guava:guava:31.1-jre required by com.exasol:parquet-io-java:jar:2.0.0 -->
+                        <!-- Rationale for exclusion: We assume that the attacker does not have access to the machine. -->
                         <exclude>sonatype-2020-0926</exclude>
```
Comment (redcatbear) [inline pom.xml:167]: You can force higher versions of the transitive dependencies in your POM.

**Review by redcatbear — DISMISSED** [review]

File: pom.xml (hunk 3 above)
Comment (ckunki) [reply pom.xml:167]: Yes, tried that, but vulnerability remained, unfortunately.

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**